### ISSUE TITLE
Hearing impaired icon

### ISSUE THREAD

**Issue by powerhcm8** [Templarian/MaterialDesign#3016]
I made a hearing impaired icon and also made a inverted the mdi-ear for consistency.

![mdi_hearing-53](https://user-images.githubusercontent.com/15136037/38402638-f6c0c5e0-3933-11e8-85bd-7dd38db473ba.png)

[hearing impaired.zip](https://github.com/Templarian/MaterialDesign/files/1882615/hearing.impaired.zip)

**Comment by CoDEmanX** [comment]
Seems like one of the cases where an enlarged slash would be justified.

**Comment by JamesCoyle** [comment]
Yeah it needs adjusting to be better centered visually over the icon

**Comment by henriquecm** [comment]
I messed up the vertical alignment before, I fixed that.
In the original, only the ear is centered and I accidentally center the whole icon.

![hearing-54](https://user-images.githubusercontent.com/15136037/38649497-5b9a1a14-3dcd-11e8-835d-dd6e42080e45.png)

**Comment by CoDEmanX** [comment]
The slash should still be placed differently IMO. Maybe 1px up and left?

**Comment by henriquecm** [comment]
Adding the alias `hearing-impaired` would be good, that's will probably be the main use of the icon.

**Comment by PeterShaggyNoble** [comment]
Alias added.
